### ISSUE TITLE
MD037/no-space-in-emphasis: Warning 

### ISSUE THREAD

**Issue by tobse80** [DavidAnson/markdownlint#354]
### Description
Look at e.g. the following line:
```markdown
some_snake_case_funcion() is _fixed_
```

The line is rendered correctly, but markdown triggers a "MD037/no-space-in-emphasis" warning for the `_funcion() is _` part:

![image](https://user-images.githubusercontent.com/26721935/101749891-8ee55400-3ace-11eb-9c61-534b07a48b7a.png)

### Expected behaviour
No MD037 warning, as there is no such "space in emphasis" situation.

### Affected version
markdownlint v0.38.0

### Additional info
Visual Studio Code
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:34:32.027Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.16299

**Comment by DavidAnson** [comment]
This appears to be a duplicate of https://github.com/DavidAnson/markdownlint/issues/324 which includes a few workarounds.
